posthog/migrations/0360_add_plugin_config_ui_fields.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.19 on 2023-11-03 12:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0359_team_external_data_workspace_id"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="deleted",
+            field=models.BooleanField(blank=True, default=False, null=True),
+        ),
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="description",
+            field=models.CharField(blank=True, max_length=400, null=True),
+        ),
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="name",
+            field=models.CharField(blank=True, max_length=400, null=True),
```

Comment:
nit: Does it make sense that `description` is the same length as `name`?

Reply:
Agreed, I copied that from experiments/insights, but I see that cohorts already use 1000, which seems more reasonable